### ISSUE TITLE
Filtering is broken for formatted combobox example

### ISSUE THREAD

**Issue by dhdaines** [w3c/aria-practices#3027]
Because the formatting done here https://github.com/w3c/aria-practices/blob/39fb654e5146151ee70675af6ae755a72d8f7ff4/content/shared/js/examples.js#L122 adds spaces around the content inside each `<li>` element in the option list, the filtering in the JavaScript, which uses this as the content to compare to:

```js
  getLowercaseContent(node) {
    return node.textContent.toLowerCase();
  }
```

does not actually work when using [the formatted HTML as shown on the web page](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/#sc1_label).

I suggest adding `trim()` to the code above so that it will work.

**Comment by dhdaines** [comment]
Thanks!  I definitely do not want to use CodePen or any such browser-based tool for development, so it is useful to have the source on the page, if it's not too much trouble.  Sometimes you also just want to look at the source and not necessarily copy/paste it anywhere, but it should also function when copied/pasted.

**Comment by mcking65** [comment]
Resolved by #3041.

@dhdaines , we made it easier to copy the source from codepen by putting a codepen button next to the source. To copy, just open codepen, focus the html, select all, and copy.

Changing the code that formats the source for display so that it could be both reliably formatted for readability and copyable from the web page in any browser on every example page would be a large testing lift that we can't contain.

The updates to support this use case are planned for publication on August 6.
